--- PHP-CS-Fixer-1.11.1/php-cs-fixer~	2016-01-20 21:00:28.000000000 +0200
+++ PHP-CS-Fixer-1.11.1/php-cs-fixer	2016-02-06 15:15:10.656768547 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env php
+#!/usr/bin/php
 <?php
 
 /*
@@ -35,7 +35,7 @@
 if (file_exists($a = __DIR__.'/../../autoload.php')) {
     require_once $a;
 } else {
-    require_once __DIR__.'/vendor/autoload.php';
+    require '/usr/share/php/Symfony/CS/autoload.php';
 }
 
 use Symfony\CS\Console\Application;
